examples/lock-app/esp32/sdkconfig.defaults.esp32c6

```diff
@@ -18,13 +18,13 @@ CONFIG_BT_ENABLED=y
 CONFIG_BT_NIMBLE_ENABLED=y
 CONFIG_BT_NIMBLE_EXT_ADV=n
 CONFIG_BT_NIMBLE_HCI_EVT_BUF_SIZE=70
-CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING=n
+CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING=y
 
 # Disable OpenThread
 CONFIG_OPENTHREAD_ENABLED=n
 
 # Disable lwip ipv6 autoconfig
-CONFIG_LWIP_IPV6_AUTOCONFIG=n
+CONFIG_LWIP_IPV6_AUTOCONFIG=y
```

Comment:
This does not appear desired. Won't this prevent IPv6 SLAAC?

Reply:
Thanks for response, comment updated.